### PR TITLE
Create What's new for April

### PR DESCRIPTION
**Pull request by BillWagner**
Run the what's new tool

### CODE REVIEW

File: docs/whats-new/2021-04.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- [Patterns (C# reference)](../csharp/language-reference/operators/patterns.md) - Added C# patterns reference
+- [Unsafe code and pointer types](../csharp/language-reference/unsafe-code.md) - Create unsafe code section - include information on function pointers
+
+### Updated articles
+
+- [C# Keywords](../csharp/language-reference/keywords/index.md) - Add unmanaged and managed function pointer call keywords
+- [Recommended tags for documentation comments (C# programming guide)](../csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments.md) - C# Metadata sweep
+- [Tutorial: Make HTTP requests in a .NET console app using C\#](../csharp/tutorials/console-webapiclient.md) - Edit pass on REST tutorial
+- [What's new in C# 9.0](../csharp/whats-new/csharp-9.md) - C# 9 what's new edit pass
+- [C# Coding Conventions (C# Programming Guide)](../csharp/programming-guide/inside-a-program/coding-conventions.md) - Move code comments to article text
```
Comment (BillWagner) [inline docs/whats-new/2021-04.md:56]: @tdykstra Do any of these meet the bar in your mind?

Comment (tdykstra) [reply docs/whats-new/2021-04.md:56]: No, I would omit from the docs what's new; most are primarily refactoring existing text, not introducing much new information.

File: docs/whats-new/2021-04.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+- [Performance counters in .NET Framework](../framework/debug-trace-profile/performance-counters.md) - Fix ms.topic for debug-trace-profile folder
+- [How to: Configure .NET Framework-Based COM Components for Registration-Free Activation](../framework/interop/configure-net-framework-based-com-components-for-reg.md) - Set ms.topic to how-to
+- [How to: Implement a Client Application that Uses the Discovery Proxy to Find a Service](../framework/wcf/feature-details/client-app-discovery-proxy-to-find-a-service.md) - Set ms.topic to how-to
+
+## .NET fundamentals
+
+### New articles
+
+- [SYSLIB0013: EscapeUriString is obsolete](../core/compatibility/syslib-warnings/syslib0013.md) - Add new syslib warnings
```
Comment (gewarren) [inline docs/whats-new/2021-04.md:23]: I thought you changed the tool not to include PR titles for new articles. Maybe you just removed the titles manually?

**Review by gewarren — APPROVED** [review]

File: docs/whats-new/2021-04.md (hunk 2 above)
Comment (BillWagner) [reply docs/whats-new/2021-04.md:23]: Yeah, I did it by hand. I'll update the code for this month.

File: docs/whats-new/2021-04.md (hunk 1 above)
Comment (BillWagner) [reply docs/whats-new/2021-04.md:56]: Done.

**Pull request merged**